### PR TITLE
i18n(es): Translate `add-yaml-support.mdx`

### PR DESCRIPTION
**Pull request by nabu1615**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translate `add-yaml-support.mdx`

### CODE REVIEW

File: src/content/docs/es/recipes/add-yaml-support.mdx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+title: Instalando un plugin de Vite o Rollup
+description: Aprende como puedes importar datos en YAML añadiendo un plugin de Rollup en tu proyecto.
+i18nReady: false
```
Comment (dreyfus92) [inline src/content/docs/es/recipes/add-yaml-support.mdx:4]:
```suggestion
i18nReady: true
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @nabu1615, te dejo una pequeña sugerencia.

File: src/content/docs/es/recipes/add-yaml-support.mdx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    ```js
+    import yml from './data.yml';
+    ```
+
+    :::note
+    Ahora que puedes importar datos YAML en tu proyecto de Astro, tu editor no va a añadir tipos para los datos importados. Para añadir tipos, crea o encuentra un archivo existente `*.d.ts` en la carpeta `src` de tu proyecto y añade lo siguiente:
+    ```ts title="src/files.d.ts"
+    // Especifica la extensión del archivo que quieres importar
+    declare module "*.yml" {
+      const value: any; // Añade la definición de tipo acá, si es necesario
```
Comment (Waxer59) [inline src/content/docs/es/recipes/add-yaml-support.mdx:42]:
```suggestion
      const value: any; // Agrega las definiciones de tipos aquí si así lo deseas.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

File: src/content/docs/es/recipes/add-yaml-support.mdx (hunk 1 above)
Comment (nabu1615) [reply src/content/docs/es/recipes/add-yaml-support.mdx]: Done.

File: src/content/docs/es/recipes/add-yaml-support.mdx (hunk 2 above)
Comment (nabu1615) [reply src/content/docs/es/recipes/add-yaml-support.mdx]: Thanks, it looks clearer now.

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Welcome to Team Docs, @nabu1615!

**Pull request merged**